contracts/external/proxies/LiquidatorProxyV1ForSoloMargin.sol

```diff
@@ -189,8 +167,19 @@ contract LiquidatorProxyV1ForSoloMargin is
                     constructAccountsArray(constants),
                     constructActionsArray(cache)
                 );
+
+                // increment the total value liquidated
+                totalValueLiquidated =
+                    totalValueLiquidated.add(cache.toLiquidate.mul(cache.owedPrice));
             }
         }
+
+        // revert if liquidator account does not have a lot of overhead to liquidate these pairs
+        Require.that(
+            totalValueLiquidated >= minValueLiquidated,
+            FILE,
+            "Not enough liquidatable value"
+        );
```

Comment:
can you also add `totalValueLiquidated` and `minValueLiquidated` to this revert message?

Reply:
done